google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -1038,6 +1053,7 @@ public static BlobListOption fields(BlobField... fields) {
       HOST_NAME,
       PATH_STYLE,
       VIRTUAL_HOSTED_STYLE,
+      BUCKET_BOUND_HOST_NAME,
```

Comment:
Jesse, I want to confirm that HOST_NAME can't be in the same way as BUCKET_BOUND_HOST_NAME. Is this accurate?